examples/logger.py

```diff
@@ -63,10 +64,12 @@ def __init__(self, parent: Optional[QObject] = None):
         self.logEdit = QTextEdit(self)
         self.logEdit.setReadOnly(True)
         self.clearButton = QPushButton("Clear")
+        self.levelSelector = QComboBox(self)
```

Comment:
We use the type of the widget as its variable name.
For example, `clearButton` is a push button, `logEdit` is a text edit.
Therefore, `levelBox` would be a valid candidate (you can use any other names that ends with box).

We follow this policy because it helps recognize the widget class instantaneously, and it describes the widget quite well.